spec/meilisearch/client/indexes_spec.rb

```diff
@@ -46,9 +73,9 @@
   it 'gets list of indexes' do
     response = @client.indexes
     expect(response).to be_a(Array)
-    expect(response.count).to eq(3)
+    expect(response.count).to eq(5)
     uids = response.map { |elem| elem['uid'] }
-    expect(uids).to contain_exactly(@uid1, @uid2, @uid3)
+    expect(uids).to contain_exactly(@uid1, @uid2, @uid3, @uid4, @uid5)
```

Comment:
Does the order matter? This is where we had some problems in Python, because sometimes MeiliSearch was sending the 5 elements but not in the same order, I don't know if this can be an issue here!

Reply:
No, the `contain_exactly` method only checks if it contains exactly the elements, not their order.